crates/l2/Makefile

```
@@ -135,8 +135,8 @@ update-system-contracts:
 # L2
 init-l2: init-metrics init-l2-no-metrics ## 🚀 Initializes an L2 Lambda ethrex Client with metrics
 
-DEFAULT_BRIDGE_ADDRESS=0x554a14cd047c485b3ac3edbd9fbb373d6f84ad3f
-DEFAULT_ON_CHAIN_PROPOSER_ADDRESS=0xc66617ef7eff0ce31a54067183f541f52f39a7dc
+DEFAULT_BRIDGE_ADDRESS=$$((grep ETHREX_WATCHER_BRIDGE_ADDRESS .env || echo 0x07a51433fc034bdcb54a053ee858e2eac9bc4974) | cut -d= -f2)
+DEFAULT_ON_CHAIN_PROPOSER_ADDRESS=$$((grep ETHREX_COMMITTER_ON_CHAIN_PROPOSER_ADDRESS .env || echo 0x7eb9ed9dd9543f88ab287ca14b912cd9c9fb2afe) | cut -d= -f2)
```

A couple of points here:
- Make a separate PR for this. 
- Instead of keeping a `DEFAULT_<CONTRACT>_ADDRESS`, I'd rather add `bridge-address` and `on-chain-proposer` with the `grep` logic there, or if you want to avoid adding more targets, we can simply add the `grep` logic immediately after the flag in `init-l2`. In addition to this, I'd avoid having a default address.

`init-l2` should work both as part of the `init` pipeline, for which I'm in favor of having the `grep` logic, and it should init the L2 node individually, for which in this case I'd prefer not having default values (as we need to maintain them). It'd be better for the second use case to enforce the user to set the contract addresses manually to avoid starting the node with an invalid state (with potentially wrong contract addresses). For this part I suggest:

```suggestion
DEFAULT_BRIDGE_ADDRESS=$$((grep ETHREX_WATCHER_BRIDGE_ADDRESS .env || echo "") | cut -d= -f2)
DEFAULT_ON_CHAIN_PROPOSER_ADDRESS=$$((grep ETHREX_COMMITTER_ON_CHAIN_PROPOSER_ADDRESS .env || echo "") | cut -d= -f2)
```

---

Done and brought `main` in [592b0f3](https://github.com/lambdaclass/ethrex/pull/2660/commits/592b0f363ce4c57079278d9344d50fd739af75b1)